lib/src/klio/transforms/helpers.py

```diff
@@ -46,6 +51,19 @@ class KlioFilterPing(
     _helpers._KlioInputDataMixin, _helpers._KlioBaseDataExistenceCheck
 ):
     """Klio transform to tag outputs if in ping mode or not."""
+    stub_config = _StubDataConfig(ping=False, force=False)
+
+    @property
+    def _data_config(self):
+        try:
+            return super()._data_config()
```

Comment:
Shouldn't this be `super()._data_config` (without the parentheses)?

I think this is causing this error in the integration tests:

```
    File "/usr/local/lib/python3.6/site-packages/klio/transforms/helpers.py", line 59, in _data_config
      return super()._data_config()
  TypeError: 'KlioGCSInputDataConfig' object is not callable```

Reply:
Yeah I'm a derp 🤦🏻‍♀️